test/task_card.test.js

```diff
@@ -1,7 +1,7 @@
 const Application = require('spectron').Application;
 const assert = require('assert');
 
-describe('Application launch', function () {
+describe('Spectron tests', function () {
```

Comment:
This shouldn't have the name of the framework we are using, but describe what are we testing